cmd/photobak/main.go

```diff
@@ -33,6 +34,7 @@ var (
 func init() {
 	flag.StringVar(&repoDir, "repo", repoDir, "The directory in which to store the downloaded media")
 	flag.BoolVar(&keepEverything, "everything", keepEverything, "Whether to store all metadata returned by API for each item")
+	flag.BoolVar(&skipRepoChecks, "skiprepochecks", skipRepoChecks, "Don't check items that already exist in the database")
```

Comment:
Oh yeah, I forgot about the other flags not having a hyphen. We could do just `-integrity` if that simplifies things. :smile: